spec/display_spec.rb

```diff
@@ -56,7 +56,7 @@
   describe "#announce_tie" do
     it "returns 'it's a tie' when the game has ended and resulted in a tie" do
       game = Game.new
-      game.instance_variable_set(:@board, Board.new(BoardCondition::TIED_GAME))
+      game.instance_variable_set(:@board, Board.new(tied_game))
```

Comment:
Hmm. Calling this method in production code would get an automatic push back (except in very special circumstances). I would encourage you to find a different way to inject the board into the game without using meta-programming.

Reply:
Resolved #39